### PR TITLE
Add License Key parameter & Easy Installation

### PR DESCRIPTION
**Pull request by muratugureminoglu**

### CODE REVIEW

**Comment by mekya** [comment]
The expection from this development is like that
```
- if ANT_MEDIA_SERVER_ZIP_FILE is not specified {
    - if there is no license key {
       - Download and Install ant media server community through github
             #You can get the latest zip file with this command `$(curl -H "Accept: application/vnd.github+json" https://api.github.com/repos/ant-media/Ant-Media-Server/releases/latest | jq -r '.assets[0].browser_download_url')`
     }
     else if license key valid {
       - Download and install ant media server enterprise through the response of `check_license` 
     }
     else {
       - Exit the installation 
     }
}
else if ANT_MEDIA_SERVER_ZIP_FILE is specified {
   - Install
}
```

**Review by mekya — CHANGES_REQUESTED** [review]
Please take a look at the comment in the PR

**Comment by muratugureminoglu** [comment]
> Please take a look at the comment in the PR

Thank you for your suggestion. A new commit has been sent.

File: .github/workflows/install-latest-to-ubuntu-20-04.yml [hunk 1]
```diff
@@ -8,8 +8,47 @@ jobs:
...
+      - run: cat /usr/local/antmedia/log/antmedia-error.log
+  Auto-Install-Enterprise-:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
+      #Get latest version
+      - run: wget https://raw.githubusercontent.com/ant-media/Scripts/4ce6a9b5fe7e1f226178d84d26e2883289e3c482/install_ant-media-server.sh && chmod 777 install_ant-media-server.sh
+      - run: wget -O ant-media-server-community.zip "https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=io.antmedia&a=ant-media-server&v=LATEST&c=community&e=zip";
```
Comment (mekya) [inline .github/workflows/install-latest-to-ubuntu-20-04.yml:50]: Why are we getting community edition in enterprise test?

File: .github/workflows/install-latest-to-ubuntu-20-04.yml [hunk 2]
```diff
@@ -8,8 +8,47 @@ jobs:
         uses: actions/checkout@v3
       - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
       #Get latest version
+      - run: wget https://raw.githubusercontent.com/ant-media/Scripts/4ce6a9b5fe7e1f226178d84d26e2883289e3c482/install_ant-media-server.sh && chmod 777 install_ant-media-server.sh
```
Comment (mekya) [inline .github/workflows/install-latest-to-ubuntu-20-04.yml:11]: why don't we get from master branch or the branch having the same name?

There is a solution for this problem in travis side.
https://github.com/ant-media/Ant-Media-Server/blob/master/.travis.yml#L106

**Review by mekya — CHANGES_REQUESTED** [review]
I've some comments. Could you please check it?

**Comment by mekya** [comment]
Please check the sonatype-lift issues as well

**Review by mekya — APPROVED** [review]

**Review by mekya — APPROVED** [review]
Thank you. Good work

File: install_ant-media-server.sh [hunk 3]
```diff
@@ -386,8 +407,13 @@ if [ "$INSTALL_SERVICE" == "true" ]; then
   check
 fi
 
-if [ $? -eq 0 ]; then
-  if [ $SAVE_SETTINGS == "true" ]; then
+# set the license key
+if [ -n "${LICENSE_KEY}" ]; then
+  sed -i "$SED_COMPATIBILITY" 's/server.licence_key=.*/server.licence_key='$LICENSE_KEY'/' "$AMS_BASE/conf/red5.properties"
```
[the remark this answers is not in the document]
Comment (mekya) [reply install_ant-media-server.sh]: @sonatype ignoreall

**Pull request merged**